src/libmongoc/tests/json/change_streams/legacy/change-streams-resume-allowlist.json

```diff
@@ -777,7 +778,7 @@
       }
     },
     {
-      "description": "change stream resumes after NotMaster",
+      "description": "change stream resumes after NotWritablePrimary",
```

Comment:
+1, I'm ok with this, although I thought that these kinds of changes were coming as part of a separate ticket. (Fine by me to fix it while we're here!)

Reply:
I missed this in my prior responses. This is a resync of these tests from the specifications repo. Testing change stream spec tests is a requirement for load balancer tests. They were updated for load balancer. The particular change of `NotMaster` => `NotWritablePrimary` comes from another spec test change unrelated to load balancers. But I resync the entire file up to the necessary commit. It may pull in unrelated changes like this, but the alternative is manually applying just the changes you are interested in, and diverging from the files from the specifications repo.